### PR TITLE
Voiceflow Integration

### PR DESCRIPTION
**Pull request by krishnacheemalapati**
Updated the code to handle our intent creation on VoiceFlow.

### CODE REVIEW

File: client/src/pages/Dashboard/components/ModalReact.js [hunk 1]
```diff
@@ -16,15 +18,18 @@ function ModalReact({ setOpenModal, intentName, intentFrequency, intentAssociate
...
         <Button variant="secondary" onClick={() => {
-              setOpenModal(false);
-            }}>Close</Button>
+          setOpenModal(false);
+        }}>Close</Button>
         <Button variant="primary" onClick={() => {
-              // DIRECT US TO VOICEFLOW HERE
-              // use {intentName}
-            }}>Add Intent to Voiceflow</Button>
+          // DIRECT US TO VOICEFLOW HERE
```
Comment (Dhairya-Khara) [inline client/src/pages/Dashboard/components/ModalReact.js:24]: No need for the comment, lets remove it.

File: client/src/pages/Dashboard/components/utils/makeBlockOnVoiceflow.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
+import axios from "axios"
+import uniqid from 'uniqid';
+
+const addBlock = async (diagramID, email, password, intent) => {
+    const token = await login(email, password)
+    const url = `https://api.voiceflow.com/v2/diagrams/${diagramID}/nodes/bulk`
+
+    const id1 = uniqid()
+    const id2 = uniqid()
+    console.log(id1)
```
Comment (Dhairya-Khara) [inline client/src/pages/Dashboard/components/utils/makeBlockOnVoiceflow.js:10]: remove the console.log

File: client/src/pages/Dashboard/components/ModalReact.js [hunk 3]
```diff
@@ -16,15 +18,18 @@ function ModalReact({ setOpenModal, intentName, intentFrequency, intentAssociate
...
+          // use {intentName}
+          console.log('click')
+          addBlock('63728e4cdbfce712134dded9', 'kharadhairya2@gmail.com', '1J7w@r@A*k86', intentName)
```
Comment (Dhairya-Khara) [inline client/src/pages/Dashboard/components/ModalReact.js:27]: Please do not commit with the password

Comment (Dhairya-Khara) [reply client/src/pages/Dashboard/components/ModalReact.js:27]: I have changed the password for security purposes.

**Review by Dhairya-Khara — CHANGES_REQUESTED** [review]
Great work. Please be careful to not commit sensitive information like the email and password. Once something is comitted on github, we cannot remove it. Thus, I have changed the password for the voiceflow account.

File: client/src/pages/Dashboard/components/ModalReact.js (hunk 1 above)
Comment (krishnacheemalapati) [reply client/src/pages/Dashboard/components/ModalReact.js:24]: True, it doesn't accurately reflect what we're doing in this function anyway. Removed it!

File: client/src/pages/Dashboard/components/ModalReact.js (hunk 3 above)
Comment (krishnacheemalapati) [reply client/src/pages/Dashboard/components/ModalReact.js:27]: OH SHOOT! Thanks for pointing that out Dhairya! Made the changes 👍

File: client/src/pages/Dashboard/components/utils/makeBlockOnVoiceflow.js (hunk 2 above)
Comment (krishnacheemalapati) [reply client/src/pages/Dashboard/components/utils/makeBlockOnVoiceflow.js:10]: Oops, left that in for testing. Removed!

**Pull request merged**